### PR TITLE
Empty pages for the approval processes and requests lists

### PR DESCRIPTION
**Pull request by lgalis**
Screenshot from 2020-02-14 16-17-02
Screenshot from 2020-02-14 16-16-53

### CODE REVIEW

**Comment by sbuenafe-rh** [comment]
@lgalis can you put the empty state message in the table? 

For reference:
https://www.patternfly.org/v4/design-guidelines/usage-and-behavior/empty-state#alignment

**Comment by Hyperkid123** [comment]
@lgalis there are components prepared exactly for this case in the front-end component repository.

File: src/presentational-components/shared/table-empty-state.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+const TableEmptyState = ({
+  title,
+  Icon,
+  description,
+  PrimaryAction,
+  renderDescription
+}) => (
+  <EmptyTable centered
+    aria-label={ `No records` }
```
Comment (Hyperkid123) [inline src/presentational-components/shared/table-empty-state.js]: `aria-label="No records"`

**Comment by sbuenafe-rh** [comment]
@lgalis looks good, but the full pagination component is missing from the bottom of the table.

**Comment by lgalis** [comment]
@sbuenafe-rh - yes, I do have a jira card for the bottom pagination, that will be a separate PR
@Hyperkid123  - updated the aria-label parameter

**Pull request merged**